source/release-notes/3-1-6.rst

```diff
@@ -0,0 +1,35 @@
+.. index:: Crafter CMS 3.1.6 Release Notes
+
+-----------------
+Crafter CMS 3.1.6
+-----------------
+
+^^^^^^^^^^^^
+New Features
+^^^^^^^^^^^^
+
+* Added workflow / publishing support for S3 bucket storage. 
```

Comment:
Added ability to manage external assets in bucket storage (S3) including workflow and publishing mechanics. Read more about it here: LINK